README.md

```diff
@@ -7,6 +7,62 @@
 
 Handles multiple cryptocurrency exchange data feeds and returns normalized and standardized results to client registered callbacks for events like trades, book updates, ticker updates, etc. Utilizes websockets when possible, but can also poll data via REST endpoints if a websocket is not provided.
 
+## Install
```

Comment:
I'm not sure an install section is needed. Since this is a library, its only really useable by a somewhat experienced python developer already. You can also install with just pip install cryptofeed. You can install a version from a git repo with setuptools as well. If in a virtualenv of some sort, you also wouldnt need the 3 on `python3` or the `--user` flag. Some parts of this might be useful (install methods for various backends)

Reply:
I agree that people who use Cryptofeed already know how to install a Python package.
However, high-quality Python libraries explain how to install with `pip install xxxx'.
For example: https://docs.aiohttp.org/en/stable/#library-installation
We can reduce the length of the chapter *Install* or move it to 'INSTALL.md' (or even remove it)... as you wish.

Reply:
Probably best to move it to a separate file